### ISSUE TITLE
add learning rate monitor

### ISSUE THREAD

**Issue by Shreeyak** [Shreeyak/pytorch-lightning-segmentation-template#39]
Monitoring the learning rate is extremely useful when deploying a learning rate scheduler strategy.
Ref: https://pytorch-lightning.readthedocs.io/en/latest/generated/pytorch_lightning.callbacks.LearningRateMonitor.html#pytorch_lightning.callbacks.LearningRateMonitor

**Comment by Shreeyak** [comment]
Done
